src/main/resources/com/aventstack/extentreports/view/html-report/exception-view/exception-view.ftl

```diff
@@ -42,11 +42,15 @@
 								</thead>
 								<tbody>
 									<#list exception.getTestList() as test>
-									<tr>
-										<td>${ test.startTime?datetime?string["${timeStampFormat}"] }</td>
-										<td class='linked' test-id='${ test.getID() }'>${ test.hierarchicalName }</td>
-										<td><pre>${ exception.exceptionInfo.getStackTrace() }</pre></td>
-									</tr>
+										<#list test.getExceptionInfoList() as testException>
```

Comment:
why do we require a loop here? A test shouldn't have more than 1 exception, correct?